app/services/template_interpolation.rb

```diff
@@ -31,12 +31,16 @@ def variables(*fields)
 private
 
   def process(template, variables)
-    Mustache.render(template, variables)
+    Mustache.render(template_as_string(template), variables)
   end
 
   def get_variables(template)
     mustache = Mustache.new
-    mustache.template = template
+    mustache.template = template_as_string(template)
     mustache.template.tags
   end
+
+  def template_as_string(template)
+    template.is_a?(ActionText::RichText) ? template.body.to_s : template
+  end
```

Comment:
You ok with this? @jamesalmond it works ok, but feels a bit yuck - ActionText::RichText does not support to_s so you need to get the body (which is ActionText::Content) and then to_s that.